content/applications/inventory_and_mrp/inventory/shipping/setup/delivery_method.rst

```diff
@@ -1,100 +1,215 @@
 
-On the delivery, check the **Carrier Information**. The carrier is the
-chosen delivery method.
+   .. image:: delivery_method/delivery-cost-example.png
+      :align: center
+      :alt: Show example of "Based on rules" shipping method with margins configured.
+
+   When the first rule is applied, the delivery cost is $31 (20 + (0.1 * 20) + 9). When the second
+   rule is applied, the delivery cost is $64 (50 + (0.1 * 50) + 9).
+
+.. _inventory/shipping/pickup:
+
+Pickup in store
+---------------
+
+To configure in-store pickup, select :guilabel:`Pickup in store` in the :guilabel:`Provider` field
+and specify the pickup location in :guilabel:`Warehouse`.
+
+To invoice the customer for the shipping cost to the pickup location, choose the :guilabel:`Get Rate
+and Create Shipment` option in the :guilabel:`Integration Level` field. Then, pick either the
+:guilabel:`Estimated cost` or :guilabel:`Real cost` radio options in the :guilabel:`Invoicing
+Policy` field to decide whether the added shipping charge on the sales order is the precise cost
+from the shipping carrier.
+
+.. seealso::
+   :ref:`Invoice cost of shipping <inventory/shipping/invoice>`
+
+.. _inventory/shipping/sales-order:
+
+Add shipping
+============
+
+To apply the configured shipping methods to calculate shipping cost, navigate to the desired sales
+order by going to :menuselection:`Sales app --> Orders --> Orders`.
+
+On the sales order, click the :guilabel:`Add shipping` button, which opens the :guilabel:`Add a
+shipping method` pop-up window. Then, choose a :guilabel:`Shipping Method` from the list.
+
+The :guilabel:`Total Order Weight` is pre-filled based on product weights (that are defined in the
+:guilabel:`Inventory` tab for each product form). Edit the field to specify the exact weight, and
+then click :guilabel:`Add` to add the shipping method.
+
+.. note::
+   The amount defined in :guilabel:`Total Order Weight` overwrites the total product weights defined
+   on the product form.
+
+The shipping cost is added to the *sales order line* as the :guilabel:`Delivery Product` detailed on
+the shipping method form.
+
+.. example::
+   `Furniture Delivery`, a delivery product with a fixed rate of `$200`, is added to sales order
+   `S00088`.
+
+     .. image:: delivery_method/delivery-product.png
+        :align: center
+        :alt: Show delivery order on the sales order line.
+
+Delivery order
+--------------
+
+The shipping method added to the sales order is linked to the shipping carrier details on the
+delivery order. To add or change the delivery method on the delivery itself, go to the
+:guilabel:`Additional Info` tab and modify the :guilabel:`Carrier` field.
+
+.. image:: delivery_method/delivery-order.png
+   :align: center
+   :alt: Shipping carrier information on the delivery form.
 
 .. seealso::
-    * :doc:`third_party_shipper`
-    * :doc:`../operation/invoicing`
+   - :ref:`Configure third-party shipping <inventory/shipping/third_party>`
```

Comment:
not sure this is needed?
it's linked I think about 4 or 5 times already in the doc...